src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.companieshouse.pscstatementdataapi.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.api.psc.Statement;
+import uk.gov.companieshouse.pscstatementdataapi.services.RetrievePscStatementService;
+
+@RestController
+public class PscStatementController {
+
+    @Autowired
+    private Logger logger;
+    @Autowired
+    private RetrievePscStatementService retrievePscStatementService;
+
+    @GetMapping("/psc-statements/{company_number}/{statement_id}")
```

Comment:
This getmapping needs to be /company/{company_number}/persons-with-significant-control-statement/{statement_id} in order to replace the already existing GET request